pkg/cloudprovider/fake/controller-kwok/charts/README.md.gotmpl

```diff
@@ -0,0 +1,28 @@
+{{ template "chart.header" . }}
+{{ template "chart.description" . }}
+
+{{ template "chart.versionBadge" . }}{{ template "chart.typeBadge" . }}{{ template "chart.appVersionBadge" . }}
+
+## Documentation
+
+For full Karpenter documentation please checkout [https://karpenter.sh](https://karpenter.sh/docs/).
+
+## Installing the Chart
+
+You can follow the detailed installation instruction in the [documentation](https://karpenter.sh/docs/getting-started/getting-started-with-karpenter) which covers the Karpenter prerequisites and installation options. The outcome of these instructions should result in something like the following command.
+
+```bash
+helm upgrade --install --namespace karpenter --create-namespace \
+  karpenter oci://public.ecr.aws/karpenter/{{ template "chart.name" . }} \
+  --version v{{ template "chart.version" . }} \
+  --set "serviceAccount.annotations.eks\.amazonaws\.com/role-arn=${KARPENTER_IAM_ROLE_ARN}" \
+  --set settings.clusterName=${CLUSTER_NAME} \
```

Comment:
Some of these settings should be dropped since they are specific to AWS

Reply:
Good call, I hadn't fixed the charts yet.